ironfish-graph-explorer/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "graphviewer",
```

not sure if this is worth changing or not
```suggestion
  "name": "ironfish-graph-explorer",
```